tools/llm_bench/benchmark.py

```diff
@@ -313,9 +316,14 @@ def main():
             iter_data_list, pretrain_time, iter_timestamp = CASE_TO_BENCH[model_args['use_case'].task](
                 model_path, framework, args.device, args.tokens_len, args.streaming, model_args,
                 args.num_iters, memory_data_collector)
+        elif model_args['use_case'].task == "visual_text_gen":
+            iter_data_list, pretrain_time, iter_timestamp = CASE_TO_BENCH[model_args['use_case'].task](
+                model_path, framework, args.device, model_args, args.num_iters,
+                memory_data_collector, decym_frames=args.video_frames)
```

Comment:
please, move video_frames to analyzed_args like videos https://github.com/krzyczar/openvino.genai/blob/f9b317ce533a97f6344b4a8674d1099d317964b8/tools/llm_bench/llm_bench_utils/model_utils.py#L121
`model_args['decym_frames'] = args.video_frames`